### ISSUE TITLE
Player endpoint returning 500 error for specific tag

### ISSUE THREAD

**Issue by jimmyk3008** [cr-api/cr-api#221]
## Request Details:
```
GET
http://api.cr-api.com/player/82CCJLUJ
```

## Response Details:
```
Code: 500

 +"error": true
  +"status": 500
  +"message": "Cannot read property 'name' of undefined | Additional information and support: http://discord.me/cr_api | Mention error ID undefined"

```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there? Recently (was fine 24 hours ago)
 2. Expected behaviour: return player profile data

**Comment by jimmyk3008** [comment]
more tags: 2GVGJPJVL, YCVU0UJ2

**Comment by jimmyk3008** [comment]
another: 82CCJLUJ

can't see similarities between the profiles

**Comment by jimmyk3008** [comment]
Taky's issues with profile tags: 2Y00GP90J , YUYLULUC , 8PCLPPR20 , 2PCPPYV8C , 2QP0U99R2 , 88CR88CQR , 2CL0VPGYR , 8RJ0JYPV8 , 8R8R9UPYP

**Comment by jimmyk3008** [comment]
Selfish fixed with deployment 8.4.33

**Comment by selfish** [comment]
@jimmyk3008 - I love how you were acting as a war-room for this issue, and carried it till the event was over.
Thank for the report. :)
